src/api/operations/create_prediction.py

```diff
@@ -31,20 +38,36 @@ def get_api_key(api_key_attempt: str = Security(api_key_header)) -> str:
 
 prediction_router = APIRouter(prefix='/air-quality')
 
-@prediction_router.post('', response_model=PostAirQualityResponse, status_code=201)
-async def predict_air_quality(airport: Airport, api_key: str = Security(get_api_key)) -> JSONResponse: # pylint: disable=unused-argument
 
+@prediction_router.on_event('startup')
```

Comment:
Did you decide against using lifespan events?

Reply:
Yeah it wasn't working :')